cap-primitives/src/rustix/fs/stat_unchecked.rs

```diff
@@ -3,6 +3,14 @@ use rustix::fs::{statat, AtFlags};
 use std::path::Path;
 use std::{fs, io};
 
+// TODO: update all these to
+// #[cfg(any(target_os = "android", target_os = "linux"))]
+// once we're on restix >= v0.34.3.
```

Comment:
Any idea when the rustix version will be updated? I don't have a sense of how big that is, so I didn't want to add it to this PR.

Reply:
The biggest thing for cap-std is that there are some semver incompatibilities between versions of io-lifetimes, which is tracking changes from [I/O safety stabilization](https://github.com/rust-lang/rust/issues/87074). Fortunately, the FCP looks to be almost done. Once it's done, I plan to do an io-lifetimes release and then a [rustix 0.35 release](https://github.com/bytecodealliance/rustix/issues/313), and then update cap-std and various other things.